python/pyspark/sql/functions.py

```diff
@@ -2050,7 +2050,6 @@ def map_values(col):
 
 # ---------------------------- User Defined Function ----------------------------------
 
-@since(2.3)
```

Comment:
We have to remove this because with python2.7, you cannot rewrite the docstring for classes.